coldfront/plugins/qumulo/utils/acl_allocations.py

```diff
@@ -37,45 +37,21 @@ def add_allocation_users(self, allocation: Allocation, wustlkeys: list):
             )
 
     @staticmethod
-    def add_user_to_access_allocation(username: str, allocation: Allocation):
+    def add_user_to_access_allocation(
+        username: str, allocation: Allocation, is_group: bool = False
+    ):
         # NOTE - just need to provide the proper username
         # post_save handler will retrieve email, given/surname, etc.
         user_tuple = User.objects.get_or_create(username=username)
+        user_tuple[0].userprofile.is_group = is_group
+        user_tuple[0].save()
 
         AllocationUser.objects.create(
             allocation=allocation,
             user=user_tuple[0],
             status=AllocationUserStatusChoice.objects.get(name="Active"),
         )
 
-    def create_acl_allocation(
-        self, acl_type: str, users: list, active_directory_api=None
-    ):
-        allocation = Allocation.objects.create(
-            project=self.project_pk,
-            justification="",
-            quantity=1,
-            status=AllocationStatusChoice.objects.get(name="Active"),
-        )
-
-        resource = Resource.objects.get(name=acl_type)
-
-        allocation.resources.add(resource)
-
-        self.add_allocation_users(allocation=allocation, wustlkeys=users)
-        self.set_allocation_attributes(
-            allocation=allocation, acl_type=acl_type, wustlkey=users[0]
-        )
-
-        try:
-            self.create_ad_group_and_add_users(
-                wustlkeys=users,
```

Comment:
There's likely to be a conflict with Santiago's code since he moved a lot of the allocation handling to a new service class.

Reply:
No conflicts and all the tests pass as of the moment, after merging in from main.  This was totally dead code which is why I removed it, and I don't believe Santiago made use of it either.  Will QA test to verify.